code/modules/mob/living/simple_animal/hostile/headcrab.dm

```diff
@@ -25,7 +25,9 @@
 	var/list/human_overlays = list()
 
 /mob/living/simple_animal/hostile/headcrab/Life(seconds, times_fired)
-	if(!is_zombie && isturf(src.loc) && stat != DEAD)
+	if(stat == DEAD)
+		return
```

Comment:
Won't this cause ..() at the end to never run?
Won't that cause issues with any other code that runs in Life()?

Reply:
You're right. One sec.